casper/src/main/scala/coop/rchain/casper/engine/Initializing.scala

```diff
@@ -110,6 +110,7 @@ class Initializing[F[_]
               )
             )
 
+        _ <- insertIntoBlockAndDagStore[F](block, approvedBlock)
```

Comment:
This is a fix for node to persist approved block, it has been broken in this commit https://github.com/rchain/rchain/pull/3092/commits/4fece59ad2d975a169f77e6a1b45ad552236d58c#diff-9a848deda385b771b42f7ce666dd6487L101